examples/proto/gen/typescript/einride/example/syntax/v2/index.ts

```diff
@@ -1,171 +1,172 @@
 // Code generated by protoc-gen-typescript-http. DO NOT EDIT.
+/* eslint-disable camelcase */
```

Comment:
Is it the method names that should be camelcase?
If that is the case I think we could make a breaking change now, and remove this disable.

Reply:
No, In this case, it's the enum `googletype_LatLng`

Reply:
gotcha! then it is better to ignore.
For context, `_` is used to ensure no duplicate identifiers.